src/sass/messaging/partials/_message-compose.scss

```diff
@@ -288,6 +286,17 @@ button.msg-btn-save {
     top: 0;
     width: auto;
   }
+
+  @media (max-width: $medium-screen) {
+    -webkit-flex: 0 0;
```

Comment:
Yeah, this has come up before, but I don't remember what the actual resolution was. Just added it for consistency, but would like to have auto-prefixing and remove the prefixes that were added manually.